lib/dealership.rb

```diff
@@ -0,0 +1,13 @@
+class Dealership
+  attr_reader :inventory
+  def initialize(dealership, address)
+    @dealership = dealership
+    @address = address
+    @inventory = []
+  end
```

Comment:
It could be a little confusing to use `dealership` as a variable here. I might use `name` instead
```suggestion
  def initialize(name, address)
    @name = name
    @address = address
    @inventory = []
  end
```